Netbox_CSV_Read/CSV/csv_to_dict.py

```diff
@@ -0,0 +1,32 @@
+import pandas as pd
+
+
+class CSVtoDict:
+
```

Comment:
Could we add a docstring to describe what this class does?

Reply:
I've added a brief Docstring.